core/src/services/oss/core.rs

```diff
@@ -213,28 +213,11 @@ impl OssCore {
     /// # Notes
     ///
     /// before return the user defined metadata, we'll strip the user_metadata_prefix from the key
-    pub fn parse_metadata(
-        &self,
-        path: &str,
-        user_metadata_prefix: &str,
-        headers: &HeaderMap,
-    ) -> Result<Metadata> {
+    pub fn parse_metadata(&self, path: &str, headers: &HeaderMap) -> Result<Metadata> {
```

Comment:
> I think we should unify this in order to be consistent.

Hi, it's better not to do this since every service has its own functions. You'll see if you delve into the implementation of different services.

The `parse_metadata` provided by `raw::http` will only parse the HTTP standards. Services may implement their own logic to meet their specific needs.